rosserial_cortex/nodes/serial_node.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
```

Comment:
Is there any difference between this and the rosserial_arduino version? https://github.com/ros-drivers/rosserial/blob/jade-devel/rosserial_arduino/nodes/serial_node.py#L1 if not we should reuse it. And if so look at parameterizing it for reuse.